assemble.sh

```diff
@@ -8,15 +8,39 @@ crate=riscv
 rm -f bin/*.a
 
 riscv64-unknown-elf-gcc -c -mabi=ilp32 -march=rv32i asm.S -o bin/$crate.o
-ar crs bin/riscv32i-unknown-none-elf.a bin/$crate.o
+ar crs bin/riscv32i-unknown-unknown-elf.a bin/$crate.o
```

Comment:
`unknown-unknown`? :thinking:

Reply:
Oops - the compiler is called `riscv64-none-elf-gcc` on my system - seems my find/replace didn't work quite as intended